pmd-cataloger/src/main/java/sfdc/sfdx/scanner/pmd/PmdRuleCataloger.java

```diff
@@ -48,22 +56,25 @@
    * stores them in a JSON file.
    */
   void catalogRules() {
+    //Check for custom rules and process them
+
     // STEP 1: Identify all of the ruleset and category files for each language we're looking at.
-    for (String language : this.languages) {
-      this.mapFilePathsByLanguage(language);
-    }
+    addPmdRules();
```

Comment:
I feel like it should be possible to write our code so it treats PMD's JARs and custom JARs identically, instead of needing to call separate methods for each.
What if we had the JS code that invokes the Java create lists of JARs for each language, and those lists would contain both the standard and custom JARs? Then we could just treat every JAR identically. Maybe that's a dumb idea, I'm not sure.